optik/echidna/interface.py

```diff
@@ -178,8 +178,15 @@ def load_tx(tx: Dict, tx_name: str = "") -> AbstractTx:
     ctx.set(sender.name, int(tx["_src"], 16), sender.size)
 
     # Translate message value
-    value = Var(256, f"{tx_name}_value")
-    ctx.set(value.name, int(tx["_value"], 16), value.size)
+    value_key = f"{tx_name}_value"
+    # Echidna will only send non-zero msg.value to payable funcs
+    # so we only make an abstract value in that case
+    # TODO: consider analyzing if funcs read msg.value
```

Comment:
```suggestion
```

If the function doesn't read `msg.value` the symbolic value will never be used, so we don't really care about making it concrete instead of symbolic